docs-src/content/_index.md

```diff
@@ -2,11 +2,226 @@
+          }
+        ],
+        "startTime": "1.350s"
+      },
+      {
+        "duration": "0.080s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.040s"
+      },
+      {
+        "duration": "0.168s",
+        "arcs": [
+          {
+            "word": "IS",
+            "confidence": 0.892
+          },
+          {
+            "word": "<eps>",
+            "confidence": 0.108
+          }
+        ],
+        "startTime": "2.120s"
+      },
+      {
+        "duration": "0.010s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.288s"
+      },
+      {
+        "duration": "0.093s",
+        "arcs": [
+          {
+            "word": "A",
+            "confidence": 0.620
+          },
+          {
+            "word": "<eps>",
+            "confidence": 0.233
+          },
+          {
+            "word": "ISN'T",
+            "confidence": 0.108
+          },
+          {
+            "word": "THE",
+            "confidence": 0.039
+          }
+        ],
+        "startTime": "2.298s"
+      },
+      {
+        "duration": "0.005s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.391s"
+      },
+      {
+        "duration": "0.273s",
+        "arcs": [
+          {
+            "word": "NEW",
+            "confidence": 0.661
+          },
+          {
+            "word": "<UNK>",
+            "confidence": 0.129
+          },
+          {
+            "word": "YOUR",
+            "confidence": 0.107
+          },
+          {
+            "word": "YOU",
+            "confidence": 0.102
+          }
+        ],
+        "startTime": "2.396s"
+      },
+      {
+        "duration": "0s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "2.670s"
+      },
+      {
+        "duration": "0.420s",
+        "arcs": [
+          {
+            "word": "DAY",
+            "confidence": 0.954
+          },
+          {
+            "word": "TODAY",
+            "confidence": 0.044
+          },
+          {
+            "word": "<UNK>",
+            "confidence": 0.002
+          }
+        ],
+        "startTime": "2.670s"
+      },
+      {
+        "duration": "0.270s",
+        "arcs": [
+          {
+            "word": "<eps>",
+            "confidence": 1.0
+          }
+        ],
+        "startTime": "3.090s"
+      }
+    ]
+  }
+}
+```
+
+## Formatted output
+
+Speech recognition systems typically output the words that were spoken, with no formatting. For example, utterances with numbers in might return “twenty seven bridges”, and “the year two thousand and three”. Cubic has the option to enable basic formatting of speech recognition results:
```

Comment:
If this is "basic formatting", what would advanced formatting be?

Reply:
Stuff like dates, times, street names, acronyms, custom formatting that a particular client has (like part numbers).